### PR TITLE
Updates NUbook for June 2021

### PR DESCRIPTION
**Pull request by ysims**
Updates:

- Members
- Wording/grammar/typos
- Broken link/s
- RoboCup 2020 and 2021
- More clarification on Getting Started recruitment task based on feedback

[Preview: Current Members](https://deploy-preview-113--nubook.netlify.app/team/current-members)
[Preview: Team History](https://deploy-preview-113--nubook.netlify.app/team/history)

### CODE REVIEW

File: src/book/01-team/01-introduction/03-current-team.mdx [hunk 1]
```diff
@@ -26,15 +26,13 @@ slug: /team/current-members
...
 ### Senior Members
 
-These members have been on the team for a number of years and have attended at least one RoboCup competition.
+These members are an expert in at least one area of the team or systems.
 
-- **Daniel Ginn**: Currently studying for a Doctorate of Philosophy in Computer Science. Specialises in localisation.
+- **Kip Hamiltons**: Undergraduate student studying a combined degree in Mathematics and Computer Science. DevOps and Testing.
 
-- **Shawn Hain**: Undergraduate student studying a combined degree in Mechanical Engineering and Mechatronics Engineering.
+- **Cameron Murtagh**: Undergraduate student studying a combined degree in Mathematics and Computer Science. NUClear and networking.
```
Comment (CMurtagh-LGTM) [inline src/book/01-team/01-introduction/03-current-team.mdx:33]: I'm not an "expert" on networking.

**Review by CMurtagh-LGTM — DISMISSED** [review]
LGTM

File: src/book/01-team/01-introduction/03-current-team.mdx (hunk 1 above)
Comment (ysims) [reply src/book/01-team/01-introduction/03-current-team.mdx:33]: Nor am I an expert on walk engines 😆 it's more what you do at NUbots. I can just make it 'NUClear'. Let me know if there's something else you'd prefer here.

File: src/book/01-team/01-introduction/03-current-team.mdx [hunk 2]
```diff
@@ -48,47 +46,37 @@ These members have been on the team for a number of years and have attended at l
...
+
+- **Moosa Hassan**: Undergraduate student studying Software Engineering. Vision.
 
-- **Samuel Dearie**: Undergraduate student studying Software Engineering. Works on adding features to NUbook.
+- **Yosiah de Koeyer**: Undergraduate student studying a combined degree in Computer Engineering and Computer Science. DevOps.
 
-- **Moosa Hassan**: Undergraduate student studying Software Engineering.
+- **Alana Noonan**: Undergraduate student studying Computer Science. Computer graphics - NUpbr and Webots model.
+
+- **Benjamin Young**: Undergraduate student studying Computer Systems Engineering. Hardware.
```
Comment (KipHamiltons) [inline src/book/01-team/01-introduction/03-current-team.mdx:61]: @xXBenjaminYoungXx you're in a combined degree now aren't you?

File: src/book/02-system/01-foundations/01-overview.mdx [hunk 3]
```diff
@@ -10,7 +10,7 @@ The NUbots repository is the main codebase that runs on the robots. Its purpose
 
 The main parts of the NUbots codebase are behaviour, motion, vision, and localisation. Behaviours include strategy, planning and skills. Motion includes kicking, getting up, and walking. Vision includes ball, goal and field detectors. Localisation determines where the robot, ball and the field are in the world through odometry.
 
-At the top level, the robot will use strategy to decide what to do, such as whether it will act as a goalkeeper or if it will act like a striker.
+At the top level, the robot will use strategy to decide what to do, such as whether it will play in the role of goalkeeper, defender or striker.
```
Comment (KipHamiltons) [inline src/book/02-system/01-foundations/01-overview.mdx:13]: Do we have capability to play defender as opposed to striker?

Comment (ysims) [reply src/book/02-system/01-foundations/01-overview.mdx:13]: Sort of. We have functionality on if we're going offensive or defensive! But not specifically defender/striker. It's more of an example. Maybe I'll try to think of a better example, particularly now that I'm having a look at behaviour.

File: src/book/01-team/01-introduction/03-current-team.mdx (hunk 2 above)
Comment (Y0sh1dk) [reply src/book/01-team/01-introduction/03-current-team.mdx:61]: This is the only place we call it "Computer Systems Engineering", thats what UON calls it, but everywhere else just calls it CE.

Comment (ysims) [reply src/book/01-team/01-introduction/03-current-team.mdx:61]: Just fixed this up

File: src/book/02-system/01-foundations/01-overview.mdx (hunk 3 above)
Comment (ysims) [reply src/book/02-system/01-foundations/01-overview.mdx]: I've changed the sentence.

**Comment by LachlanCourt** [comment]
Are we adding the result of the 2021 league in here or later? If later then LGTM

**Comment by ysims** [comment]
Results can be in a separate PR. I'd like to get this in before July.

File: src/book/01-team/02-joining/01-how-we-work.mdx [hunk 4]
```diff
@@ -24,7 +24,7 @@ We use GitHub for development and code collaboration. All NUbots code can be fou
 
 The main repositories are:
 
-- [NUbots](https://github.com/NUbots/NUbots) - monorepo with the C++ codebase that runs on the robots, and NUsight, the web-based debugging environment we used to understand and improve our systems.
+- [NUbots](https://github.com/NUbots/NUbots) - C++ codebase that runs on the robots and NUsight, the web-based debugging environment we used to understand and improve our systems.
```
Comment (Y0sh1dk) [inline src/book/01-team/02-joining/01-how-we-work.mdx:27]: Something here about our ability to run the codebase locally would be nice. Possibly:
```suggestion
- [NUbots](https://github.com/NUbots/NUbots) - C++ codebase that runs on the robots, or locally via simulation and NUsight, the web-based debugging environment we used to understand and improve our systems.
```

**Comment by ysims** [comment]
Actually I will put the results in.

**Review by Y0sh1dk — APPROVED** [review]
LGTM.

**Review by LachlanCourt — APPROVED** [review]
LGTM

File: src/book/01-team/01-introduction/03-current-team.mdx [hunk 5]
```diff
@@ -48,48 +46,43 @@ These members have been on the team for a number of years and have attended at l
...
 - Bryce Tuppurainen, Software Engineering
-- Benjamin Young, Computer Systems Engineering
 
 ## Supporting Members
 
 Past members of the team that are still involved with projects, mentoring, and support.
 
-- **Matt Amos**: Computer Science and Computer Engineering Honours graduate. Specialises in synthetic dataset generation and robotic movement.
-
-- **Brendan Annable**: Bachelor of Software Engineering Graduate. Ex-Team Leader. Lead Developer of NUsight.
```
Comment (JosephusPaye) [inline src/book/01-team/01-introduction/03-current-team.mdx:90]: Brendan is still around

Comment (JosephusPaye) [reply src/book/01-team/01-introduction/03-current-team.mdx:90]: @ysims

**Pull request merged**